roles/telemetry_autoscaling/tasks/test_services.yml

```diff
@@ -17,14 +17,27 @@
   failed_when:
     - result.stdout != "True"
 
-# - name: Verify that custom resource AUTOSCALING is ready
-# It is returning an unexpected response and needs further debugging. will raise another PR to inspect this
-#   ansible.builtin.command:
-#     cmd: |
-#       oc get autoscaling autoscaling -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
-#   register: result
-#   failed_when:
-#     - result.stdout != "True"
+- name: Check that the autoscaling CR  is ready
+  block:
+    - name: Verify that custom resource AUTOSCALING is ready
+      ansible.builtin.command:
+        cmd: |
+          oc get autoscaling autoscaling -o jsonpath='{.status.conditions[?(@.type=="Ready")].status}{"\n"}'
+      register: result
+      failed_when:
+        - result.stdout != "True"
+  rescue:
+    - name: view autoscaling objects
+      ansible.builtin.command:
+        cmd: "{{ item }}"
+      register: result
```

Comment:
can you please use a different variable names  for different tasks to avoid conflicts?
for example: register: autoscaling_result